app/api/server/v1/users.js

```diff
@@ -568,3 +568,14 @@ API.v1.addRoute('users.removePersonalAccessToken', { authRequired: true }, {
 		return API.v1.success();
 	},
 });
+
+API.v1.addRoute('users.requestDataDownload', { authRequired: true }, {
+	post() {
+		const result = Meteor.runAsUser(this.userId, () => Meteor.call('requestDataDownload', { fullExport: false }));
```

Comment:
The `fullExport` param could be an API param too, so that the client can determine if they want the full export or not.